sydent/db/threepid_associations.py

```diff
@@ -87,14 +87,14 @@ def removeAssociation(self, threepid, mxid):
                 (threepid['medium'], threepid['address'], ts),
             )
             logger.info(
-                "Deleting local assoc for %s/%s/%s replaced %d rows",
-                threepid['medium'], threepid['address'], mxid, cur.rowcount,
+                "Deleting local assoc %s. Replaced %d rows",
+                mxid, cur.rowcount,
```

Comment:
Don't we consider mxids PII?

Reply:
Oh, possibly since people could have their real names in it. Is there another way we can allow for easy database diving without using 3PID or MXID info?